07-sampling.Rmd

```diff
@@ -200,7 +200,7 @@ bowl
 
 Observe that `bowl` has 2400 rows, telling us that the bowl contains 2400 equally-sized balls. The first variable `ball_ID` is used as an *identification variable* as discussed in Subsection \@ref(identification-vs-measurement-variables); none of the balls in the actual bowl are marked with numbers. The second variable `color` indicates whether a particular virtual ball is red or white. View the contents of the bowl in RStudio's data viewer and scroll through the contents to convince yourself that `bowl` is indeed a virtual analog of the actual bowl in Figure \@ref(fig:sampling-exercise-1).
 
-Now that we have a virtual analog of our bowl, we now need a virtual analog to the shovel seen in Figure \@ref(fig:sampling-exercise-2) to generate virtual samples of 50 balls. We're going to use the `rep_sample_n()` function included in the `moderndive` package. This function allows us to take `rep`eated, or `rep`licated, `samples` of size `n`. 
+Now that we have a virtual analog of our bowl, we now need a virtual analog to the shovel seen in Figure \@ref(fig:sampling-exercise-2) to generate virtual samples of 50 balls. We're going to use the `rep_sample_n()` function included in the `infer` package. This function allows us to take `rep`eated, or `rep`licated, `samples` of size `n`. 
```

Comment:
Hey @ismayc. At this point I don't remember why we imported `infer::rep_sample_n()` into `moderndive`. Should use `rep_sample_n()` via `moderndive`, or via `infer`? If we do that latter, then we'll have to include a message to the effect of "Let's load infer, but we'll discuss it in the next Chapter". Thoughts?

Reply:
It was exactly for this reason. We don’t want to confuse students by introducing a whole new package that we won’t discuss until a later chapter. I recommend we keep it as `ModernDive` to continue building on that throughout the chapter.

Reply:
@davidkane9 could you revert the earlier change of which pkg we use? We'll then approve. Thanks for going over this with a fine toothed-comb

Reply:
Sorry. Missed this. But looks like you have fixed the other issue.

Reply:
No problem! Thanks much.